runner/src/build.rs

```diff
@@ -12,13 +12,21 @@ pub fn build(args: &BuildArgs) {
     if let Some(firmware) = &args.firmware {
         let firmware = match locate_artifact(firmware) {
             Some(Artifact::Source { name }) => build_target(Target::Firmware(name), &cfg),
+            Some(Artifact::MIRALIS) => build_target(Target::MiralisAsFirmware, &cfg),
             Some(Artifact::Downloaded { name, url }) => download_artifact(&name, &url),
             None => PathBuf::from_str(firmware).expect("Invalid firmware path"),
         };
         println!("Built firmware, binary available at:");
         println!("{}", firmware.display());
     } else {
-        let miralis = build_target(Target::Miralis, &cfg);
+        let mut target = Target::Miralis;
+        if let Some(as_firmware) = &args.as_firmware {
+            if *as_firmware {
+                target = Target::MiralisAsFirmware;
+            }
+        }
```

Comment:
Do we need that special case if we can build Miralis by passing `miralis` as the firmware name?

Reply:
You are right. I had a look at my code again and I realized it can be a bit simplified.